examples/ls.rs

```diff
@@ -24,7 +24,7 @@ fn format_file_size(size: u64) -> String {
 fn main() -> io::Result<()> {
     let file = File::open("resources/fat32.img")?;
     let buf_rdr = BufStream::new(file);
-    let fs = FileSystem::new(buf_rdr, FsOptions::new())?;
+    let fs = FileSystem::new(StdIoWrapper::new(buf_rdr), FsOptions::new())?;
```

Comment:
This is exactly why I made `IntoStorage` trait - to make it possible to pass `std::fs::File` directly. Anyway why in some places you used `into()` and in some `StdIoWrapper::new` directly? IMHO `into` way is better

Reply:
Could do something like `pub fn new(disk: impl Into<IO>, ...)` to keep making this work.